views/partials/_banner.njk

```diff
@@ -1,17 +1,20 @@
-{% if env("PULL_REQUEST") === "true" %}
-  {% set bannerTag = "Preview" %}
-  {% set bannerText %}
-    This is a preview of
-    a <a class="govuk-link" href="https://github.com/alphagov/govuk-design-system/pull/{{ env("REVIEW_ID")  }}">proposed change</a> to the
-    <a class="govuk-link" href="https://design-system.service.gov.uk">Design System</a>.
-  {% endset %}
-{% elif env("PULL_REQUEST") === "false" %}
-  {% set bannerTag = "Archive" %}
-  {% set bannerText %}
-    This is an archived version of
-    the <a class="govuk-link" href="https://github.com/alphagov/govuk-design-system/tree/{{ env("BRANCH") }}">{{ env("BRANCH") }}</a> branch for the
-    <a class="govuk-link" href="https://design-system.service.gov.uk">Design System</a>.
-  {% endset %}
+{#- Show banner for archive, preview and development builds only -#}
+{% if ["branch-deploy", "deploy-preview", "dev"].includes(env("CONTEXT")) %}
```

Comment:
I think using `CONTEXT` makes sense, but just in case they introduce additional contexts in the future, could we flip this and check for anything other than `production` instead?

```suggestion
{% if env("CONTEXT") !== "production" %}
```

https://docs.netlify.com/configure-builds/environment-variables/#build-metadata

Reply:
I'd love to do this, 2 mins

I wasn't sure if it helped self-document the code, especially with `NODE_ENV=production` too

Reply:
Alternatively, we could switch the `PULL_REQUEST` check out for context?

```
if CONTEXT == deploy-preview
  show pull request banner
elif CONTEXT == branch-deploy
  show archive banner
elif CONTEXT == dev
  show new dev banner for local development
end
```

Reply:
Done